src/pages/dmt5/_flexibility.js

```diff
@@ -126,8 +138,20 @@ const Flexibility = () => {
             </StyledHeader>
             <Flex mb="4rem" tablet_direction="column" tablet_ai="center" tablet={{ m: '0' }}>
                 {content.map((item, idx) => {
+                    let show_content = false
+
+                    if (item.show_eu !== undefined) {
+                        if(is_eu_country) {
+                            show_content = item.show_eu
+                        } else {
+                            show_content = !item.show_eu
+                        }
+                    } else {
+                        show_content = true
+                    }
+
```

Comment:
Instead of this doing nested logic here. Can we just do conditional checks such that `is_eu_country ? 
<RenderOurEuComponent /> : <RenderOurNonEuComponent />`?

Reply:
fixed